kubernetes/samples/scripts/common/wdt-and-wit-utility.sh

```diff
@@ -206,15 +210,18 @@ function run_wdt {
 
   cd $WDT_DIR || return 1
 
-  echo @@ "Info:  WDT extractDomainResource.sh output will be in extract${action}.out and $wdt_log_dir"
-
-  $wdt_bin_dir/extractDomainResource.sh \
-     -oracle_home $oracle_home \
-     -domain_resource_file domain${action}.yaml \
-     -domain_home $domain_home_dir \
-     -model_file $model_final \
-     -variable_file $inputs_final > extract${action}.out 2>&1
-
+  cmd="
+  $wdt_bin_dir/extractDomainResource.sh
+     -oracle_home $oracle_home
+     -domain_resource_file domain${action}.yaml
+     -domain_home $domain_home_dir
+     -model_file $model_final
+     -variable_file $inputs_final
```

Comment:
IIRC, the 'inputs' file has quite a few tunables. Will they all continue to work using the 'extracted domain resource' instead of the hard coded domain resource?

Reply:
This function is only used by the domain-on-pv samples and used to be in multiple create-domain-script.sh scripts. It was simply refactored to this new common utlitliy script. The logic should remain unchanged.